common/src/main/java/com/microsoft/identity/common/internal/cache/SharedPreferencesFileManager.java

```diff
@@ -151,29 +179,20 @@ public SharedPreferencesFileManager(
             final String name,
             final int operatingMode,
             final IStorageHelper storageHelper) {
-        if (operatingMode == -1 && storageHelper == null) {
-            Logger.verbose(TAG, "Init: ");
-        } else if (storageHelper == null) {
-            Logger.verbose(TAG, "Init with operating mode: " + TAG);
-        } else if (operatingMode == -1) {
-            Logger.verbose(TAG, "Init with storage helper:  " + TAG);
-        } else {
-            Logger.verbose(TAG, "Init with operating mode and storage helper " + TAG);
-        }
-        mSharedPreferencesFileName = name;
-        mSharedPreferences = context.getSharedPreferences(name, operatingMode == -1 ? Context.MODE_PRIVATE : operatingMode);
-        mStorageHelper = storageHelper;
+        this(context, name, operatingMode, null, new LruCache<String, String>(256));
```

Comment:
```suggestion
        this(context, name, operatingMode, storageHelper, new LruCache<String, String>(256));
```

I assume we don't mean to discard the provided instance?